web/client/components/mapcontrols/locate/LocateBtn.jsx

```diff
@@ -110,15 +110,22 @@ class LocateBtn extends React.Component {
         if (this.props.locate !== 'PERMISSION_DENIED' && !checkingGeoLocation && !geoLocationAllowed) {
             // check if we are allowed to use geolocation feature
             checkingGeoLocation = true;
-            navigator.geolocation.getCurrentPosition(() => {
+            if (typeof navigator.geolocation !== 'undefined') {
```

Comment:
what about this?
```suggestion
            if (navigator.geolocation) {
```

or this

```suggestion
            if (navigator?.geolocation?.getCurrentPosition) {
```